sdk/PROJECTS.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to the Daml and Canton Projects
+
+The following projects are of great value but less urgency and have therefore not yet been picked up by a developer. If 
+you are up for a challenge and would like to move the project significantly forward, please consider picking one and
+contributing to it. 
+
+The list summarizes projects that are part of the Daml ecosystem beyond the core Daml language and SDK repository.
+
+If you are interested in one of these projects and would like to know more about it, please reach out to the maintainers via https://discuss.daml.com/.
+
+## Daml
+### Public, Protected, Private Access Modifiers
+Right now, all templates and choices are public. We would like to introduce new access modifiers that restrict
+the visibility of templates and choices:
+- instances of contracts of protected templates cannot be created on the Ledger API.
+- instances of contract of private templates can only be created within the defining module.
+- protected choices cannot be exercised on the Ledger API.
+- private choices can only be exercised within the defining module.
+
+### Separate Daml Script from Daml
+Daml script is too tightly entangled with Daml. We would like to completely separate it such that Daml scripts are not a dependency of Canton.
+
+### DAR Dependency Management
+The current DAR dependency management is very basic. At compile time, all dependencies of a DAR get packaged by default.
+Instead, we would like to separate this such that by default, only "lean" DARs exist, and an additional command to package
+all dependencies together into a shipping DAR. On top of this, it would be desirable to establish a Maven like system
+to share DAR dependencies (by e.g. using Maven ...)
+
+### DAR Verification Tooling
+We would like to extend our DAR tooling to allow users to verify any DAR that they have received. The compilation of DARs
+is deterministic, but requires the same versions of the SDK. Therefore, the goal would be to build a tool that can verify
+a DAR by determining the right version of the SDK, compiling the DAR and verifying the content.
+
+## Canton
+### Core
+#### Native Canton Console
+The Canton Console is a Scala based application to interact with Canton nodes through their Admin APIs by exposing
+a Scala prompt to the user. While the experience should largely resemble a classic shell prompt, it is clear that
+this is not fully provided by the current implementation. In addition, the startup time of the JVM make bash scripting
+with the Canton console painful. Therefore, we would like to implement a native shell for the Canton Console which can
+run from the Command line without the need for the JVM. Many solutions are possible, from Rust to Python.
+
+#### Interactive Submission to Replace Daml Exceptions
```

Comment:
remove this one as we work on it?

Reply:
Right now, we are not working on replacing Daml Exceptions, no? We are just adding the crypto custody stuff.